common/models.py

```diff
@@ -1,16 +1,20 @@
 import tensorflow as tf
+from utils.misc import print_tensor_shape
 
+_default = object()
 
 class ConvNetwork:
     def __init__(self, layer_specs=None,
                  activation_fn=tf.nn.leaky_relu,
+                 last_activation_fn=_default,
```

Comment:
I've been dealing with the last activation function by appending an extra "output layer" after calling _get_conv_tower. If we update ConvNetwork to have a last_activation_fn, we need to update the other networks to use it.

Reply:
I think that it's common enough of a use case that we should have it as an option. It's not meant to be necessary and should be compatible with what you already have (i.e should just default to the value of activation_fn).